source/base/static/base/js/backbone/crawl_models.js

```diff
@@ -92,6 +92,15 @@
       var that = this;
       this.collection = collection;
       this.collectionView = collectionView;
+
+      // If the add crawl modal has been opened at the same time as the add model form,
+      // change focus back to the add crawl modal after the form has been closed
+      // or submitted.
+      $(this.modal).on('hidden.bs.modal', function(event){
+        if ($("#addCrawlModal").attr("aria-hidden") == "false"){
```

Comment:
okay, I understand more of what's going on here.  There are more general solutions to this problem, but they are probably unnecessary complications.  Let's remember to come back and refactor this if we end up needing this in more than one or two places.